molecule/default/molecule-with-systemd.yml

```
@@ -15,24 +15,18 @@ lint: |
   ansible-lint
   flake8
 platforms:
-  - cgroupns_mode: host
-    command: /lib/systemd/systemd
-    image: geerlingguy/docker-amazonlinux2-ansible:latest
-    name: amazonlinux2-systemd
-    platform: amd64
-    pre_build_image: yes
-    privileged: yes
-    volumes:
-      - /sys/fs/cgroup:/sys/fs/cgroup:rw
-  - cgroupns_mode: host
-    command: /lib/systemd/systemd
-    image: geerlingguy/docker-debian9-ansible:latest
-    name: debian9-systemd
-    platform: amd64
-    pre_build_image: yes
-    privileged: yes
-    volumes:
-      - /sys/fs/cgroup:/sys/fs/cgroup:rw
+  # Amazon Linux 2023 does not offer the following packages required by httpd:
+  # mod_auth_gssapi,mod_authnz_pam
```

We usually put a space after a comma- slightly more readable:
```suggestion
  # mod_auth_gssapi, mod_authnz_pam
```

---

59d439c has all relevant changes